components/remote/ui/ir/CardButtonsTransport.qml

```diff
@@ -29,91 +31,118 @@ Item {
-            title: qsTr("Rec") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.record(); }
-        }
-
-        Item { width: 118; height: 70 }
-
-        Button {
-            visible: obj.isSupported(Remote.F_EXIT)
-            title: qsTr("Exit") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.exit(); }
-        }
-
-        Item { width: 118; height: 70 }
-
-        Button {
-            visible: obj.isSupported(Remote.F_BACK)
-            title: qsTr("Back") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.back(); }
+        Row {
+            Layout.alignment: Qt.AlignHCenter
+            spacing: 32
+            IconButton {
+                visible: obj.isSupported(Remote.F_PREVIOUS)
+                image.source: "qrc:/images/remote/skip_previous-white-48dp.svg"
+                mouseArea.onClicked: { obj.previous(); }
+            }
+
+            IconButton {
+                visible: obj.isSupported(Remote.F_INFO)
+                image.source: "qrc:/images/remote/info-white-48dp.svg"
+                mouseArea.onClicked: { obj.info(); }
+            }
+
+            IconButton {
+                visible: obj.isSupported(Remote.F_NEXT)
+                image.source: "qrc:/images/remote/skip_next-white-48dp.svg"
+                mouseArea.onClicked: { obj.next(); }
+            }
         }
 
-        Button {
-            visible: obj.isSupported(Remote.F_HOME)
-            title: qsTr("Home") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.home(); }
+        Row {
+            Layout.alignment: Qt.AlignHCenter
+            spacing: 32
+            Button {
+                visible: obj.isSupported(Remote.F_EXIT)
+                title: qsTr("Exit") + translateHandler.emptyString
+                mouseArea.onClicked: { obj.exit(); }
+            }
+
+            Button {
+                visible: obj.isSupported(Remote.F_GUIDE)
+                title: qsTr("Guide") + translateHandler.emptyString
+                mouseArea.onClicked: { obj.guide(); }
+            }
         }
 
-        Button {
-            visible: obj.isSupported(Remote.F_MENU)
-            title: qsTr("Menu") + translateHandler.emptyString
-            mouseArea.onClicked: { obj.menu(); }
+        Row {
+            Layout.alignment: Qt.AlignHCenter
+            spacing: 32
+            Button {
+                visible: obj.isSupported(Remote.F_BACK)
+                title: qsTr("Back") + translateHandler.emptyString
+                mouseArea.onClicked: { obj.back(); }
+            }
+
+            IconButton {
+                visible: obj.isSupported(Remote.F_HOME)
+                image.source: "qrc:/images/remote/home-white-48dp.svg"
+                mouseArea.onClicked: { obj.home(); }
+            }
+
+            IconButton {
+                visible: obj.isSupported(Remote.F_MENU)
+                image.source: "qrc:/images/remote/menu-white-48dp.svg"
```

Comment:
For icons, best would be to extend the icons.tft font file with new icons and reference them from Style.qml.